autoware_map_msgs/README.md

```diff
@@ -12,6 +12,11 @@ The message contains a pointcloud data attached with an ID.
 
 The message contains a pointcloud meta data attached with an ID. These IDs are intended to be used as a query for selected PCD map loading (see `GetSelectedPointCloudMap.srv` section).
 
+## MapProjectorInfo.msg
+
+The message contains the information required to project global coordinates to local coodraintes used by Autoware, which includes the name of the projection method and the parameters for the projection.
```

Comment:
```suggestion
The message contains the information required to project global coordinates to local coordinates used by Autoware, which includes the name of the projection method and the parameters for the projection.
```

small typo

Reply:
FIxed in https://github.com/autowarefoundation/autoware_msgs/pull/102/commits/45bf1eafa0b48e87e7c5f2dc08692e9820eb8913